openerp/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -468,8 +468,14 @@ def test_60_x2many_domain(self):
         self.assertNotIn(message, discussion.important_messages)
 
         message.important = True
+        discussion.invalidate_cache()
         self.assertIn(message, discussion.important_messages)
 
+        discussion.write({
+            "important_messages": [(5,)],
+        })
+        self.assertFalse(discussion.important_messages)
+
```

Comment:
Please restore the original test, and replace that code by one on the field `very_important_messages`:
```python
# writing on very_important_messages should call its domain method
self.assertIn(message, discussion.very_important_messages)
discussion.write({'very_important_messages': [(5,)]})
self.assertFalse(discussion.very_important_messages)
self.assertFalse(message.exists())
```